api/tests/core/offers/test_api.py

```diff
@@ -344,6 +357,17 @@ def test_edit_stock(self):
         assert edited_stock.quantity == 7
         assert update_info is False
 
+    def test_edit_stock_id_at_provider(self):
+        # Given
+        existing_stock = factories.StockFactory(price=10, idAtProviders="I have a secret")
+
+        # When
+        edited_stock, _ = api.edit_stock(stock=existing_stock, id_at_provider="I'm Batman !!!")
+
+        # Then
+        assert edited_stock == models.Stock.query.filter_by(id=existing_stock.id).first()
```

Comment:
(détail / suggestion)

```suggestion
        assert edited_stock == models.Stock.query.get(existing_stock.id)
```

Reply:
J'ai une erreur bizarre de `sqlalchemy` dans ce cas : 
![image](https://github.com/pass-culture/pass-culture-main/assets/169165300/3ce2390d-c8f3-4856-bc18-93a04a2167d0)